src/util/parser-error-logger.js

```diff
@@ -1,14 +1,13 @@
 const {
   NODE_ENV,
-  REACT_APP_ENABLE_PARSER_DEBUGGING
+  REACT_APP_ENABLE_PARSER_DEBUGGING,
 } = process.env;
 
-let errorFn;
+const isDebuggingEnabled =
+  NODE_ENV === 'development' && REACT_APP_ENABLE_PARSER_DEBUGGING === 'true';
 
-if (NODE_ENV === 'development' && REACT_APP_ENABLE_PARSER_DEBUGGING === 'true') {
-  errorFn = (err) => { console.error(err); };
-} else {
-  errorFn = () => {};
-}
+const errorFn =
+  /* eslint-disable no-console */
+  isDebuggingEnabled ? (err) => { console.error(err); } : () => {};
```

Comment:
No, because we're not returning the result of an expression. Same applies for `ref`s, since you're not returning the expression (which is just assignment) you need to include curly braces